pubsub-adapter-deployment.yml

```diff
@@ -34,6 +34,24 @@ spec:
             limits:
               cpu: "0.4"
               memory: "64Mi"
+          readinessProbe:
+            exec:
+              command:
+                - cat
+                - $(READINESS_FILE_PATH)
+            initialDelaySeconds: 1
+            periodSeconds: 2
+            failureThreshold: 10
+            successThreshold: 1
+          livenessProbe:
```

Comment:
i don't think this is appropriate for a liveness check as currently the file only gets written to on startup and never touched after that, a liveness probe should be something thats constantly updated - in this case maybe touching the file every minute is good enough....

Reply:
Removed the liveness probe for the moment